code/modules/arcade/mob_hunt/mob_datums.dm

```diff
@@ -0,0 +1,549 @@
+		multiplier *= 0.5
+	else if(attack_type in secondary_type.weakness)
+		multiplier *= 2
+	else
+		multiplier *= 1
+	return multiplier
+
+/datum/mob_hunt/proc/take_damage(raw_damage, datum/mob_type/attack_type)
+	var/message = ""
+	var/multiplier = calc_dam_multiplier(attack_type)
+	var/total_damage = raw_damage * multiplier
+	if(!health[1])	//it's already downed, quit hitting it
+		return null
+	health[1] = max(health[1] - total_damage, 0)
+	switch(multiplier)
+		if(0)
+			message += "The attack is completely ineffective! "
+		if(0.25)
+			message += "It's barely effective... "
+		if(0.5)
+			message += "It's not very effective... "
+		if(2)
+			message += "It's super effective! "
+		if(4)
+			message += "It's ultra effective! "
+	if(!health[1])
+		message += "[nickname ? nickname : mob_name] is downed!"
+	return message
+
+/datum/mob_hunt/proc/levelup()
+	var/message = ""
+	level++
+	if(level > max_level)		//This is where we would trigger an evolution, when those are added (need to add evolved forms first)
+		level = max_level		//for now though, we'll just cap them back at their max_level
+		message += "[nickname ? nickname : mob_name] can't get any stronger right now!"
+	else
+
+		health[2] = base_health + (level * health_multiplier)
+		message += "[nickname ? nickname : mob_name] has reached level [level]!"
+
+/datum/mob_hunt/proc/get_type1()
+	if(!primary_type)
+		return "Typeless"
+	else
+		return initial(primary_type.name)
+
+/datum/mob_hunt/proc/get_type2()
+	if(!secondary_type)
+		return null
+	else
+		return initial(secondary_type.name)
+
+
+/datum/mob_hunt/nemabug
+	mob_name = "Nemabug"
+	run_chance = 50
+	min_level = 1
+	max_level = 10
+	primary_type = TYPE_BUG
+	icon_state_normal = "nemabug"
+	icon_state_shiny = "nemabug_shiny"
+	area_blacklist = list(/area/toxins)
+	turf_blacklist = list()
+	area_whitelist = list(/area/hydroponics,
+						/area/hallway/secondary/construction)
+	turf_whitelist = list(/turf/simulated/floor/grass)
+	lifetime = 6000
+
+/datum/mob_hunt/stoutquill
+	mob_name = "Stoutquill"
+	run_chance = 50
+	min_level = 5
+	max_level = 15
+	primary_type = TYPE_ICE
+	icon_state_normal = "stoutquill"
+	icon_state_shiny = "stoutquill_shiny"
+	area_blacklist = list(/area/maintenance/turbine,
+						/area/maintenance/incinerator,
+						/area/crew_quarters/kitchen)
+	turf_blacklist = list()
+	area_whitelist = list(/area/toxins/server_coldroom)
+	turf_whitelist = list()
+	lifetime = 4500
+
+/datum/mob_hunt/spectra
+	mob_name = "Spectra"
+	run_chance = 35
+	min_level = 1
+	max_level = 10
+	primary_type = TYPE_POISON
+	icon_state_normal = "spectra"
+	icon_state_shiny = "spectra_shiny"
+	area_blacklist = list(/area/medical,
+						/area/security/medbay,
+						/area/janitor)
+	turf_blacklist = list()
+	area_whitelist = list(/area/medical/virology,
+						/area/toxins,
+						/area/medical/research,
+						/area/medical/research_shuttle_dock,
+						/area/crew_quarters/hor,
+						/area/maintenance/asmaint2)
```

Comment:
This tabbing, why are you doing it like this? It hurts. Isn't it supposed to be more uniformed?